tico/serialize/operators/utils.py

```diff
@@ -49,3 +50,42 @@ def create_builtin_operator(
     operator.inputs = [graph.get_tid(input) for input in inputs]
     operator.outputs = [graph.get_tid(output) for output in outputs]
     return operator
+
+
+def get_integer_dtype_min(dtype_str: str):
```

Comment:
```suggestion
def get_integer_dtype_min(dtype_str: str) -> int:
```

Please add unit tests for this function. There are some cases that produce wrong outputs.

"int1" -> -1
"int0" -> -0.5

Better to use if statement rather than re to avoid such corner cases.

Reply:
Good point 👍
However, IMHO, is there any possibility for such cases like `int1` or `int0`?
If we use signed type, at leaset 2 bit is needed I guess.. 🤔

Reply:
They are "corner cases" as I mentioned :) They may rarely happen, but we shouldn't just ignore them.

Reply:
Thank you for clarification! 😄 As you said, it would better to prepare for such cases